### ISSUE TITLE
Example won't compile

### ISSUE THREAD

**Issue by LefterisJP** [PistonDevelopers/Piston-Tutorials#76]
This is the error output I am getting

```
   Compiling getting-started-spinning-square v0.0.0 (file:///home/lefteris/w/rust_test)
src/main.rs:29:13: 29:18 error: the trait `graphics::graphics::Graphics` is not implemented for the type `opengl_graphics::back_end::GlGraphics` [E0277]
src/main.rs:29             clear(GREEN, gl);
                           ^~~~~
src/main.rs:36:13: 36:22 error: the trait `graphics::graphics::Graphics` is not implemented for the type `opengl_graphics::back_end::GlGraphics` [E0277]
src/main.rs:36             rectangle(RED, square, transform, gl);
                           ^~~~~~~~~
error: aborting due to 2 previous errors
Could not compile `getting-started-spinning-square`.
```

I am running ArchLinux with rust `rustc 1.0.0-dev (built 2015-05-17)`

Any ideas what's wrong?

**Comment by bvssvni** [comment]
The dependencies are now on crates.io internally which causes this error. I'll update it.

**Comment by LefterisJP** [comment]
Thank you, this does indeed address the issue

**Comment by xaviershay** [comment]
I'm getting this trying to run the tutorial with SDL2 using latest dependencies from crates.io:

```
 "piston 0.17.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "piston2d-graphics 0.14.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "piston2d-opengl_graphics 0.23.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "piston_window 0.35.0 (registry+https://github.com/rust-lang/crates.io-index)",
 "pistoncore-sdl2_window 0.24.0 (registry+https://github.com/rust-lang/crates.io-index)",
```

Not sure if it's related, but error is the same. Going to keep poking at it.

**Comment by xaviershay** [comment]
Found it, released `piston2d-opengl_graphics` depends on 0.13 of `piston2d-graphics`: https://github.com/PistonDevelopers/opengl_graphics/blob/master/Cargo.toml#L36

Pinning to 0.13.0 in my `Cargo.toml` works around.

**Comment by xaviershay** [comment]
Relevant issue: https://github.com/PistonDevelopers/opengl_graphics/issues/222
